src/components/ui/Toggles/ToggleSwitch.jsx

```diff
@@ -1,15 +1,16 @@
-import { useState } from 'react';
-import PropTypes from 'prop-types';
+import { useState } from "react";
+import PropTypes from "prop-types";
 
 const bgLight = `bg-[url('./assets/light.svg')]`;
 const bgDark = `bg-[url('./assets/dark.svg')]`;
 
 ToggleSwitch.propTypes = {
   isOn: PropTypes.bool.isRequired,
   onChange: PropTypes.func.isRequired,
+  isDisabled: PropTypes.bool,
```

Comment:
better to use "positive" name convention - `isEnabled`

Reply:
checkbox does not show it's current value without it dunno why.